pkg/api/types.go

```diff
@@ -1489,6 +1556,9 @@ type ServiceSpec struct {
 	// those endpoints.
 	Selector map[string]string `json:"selector"`
 
+	// DisruptionBudget defines the max disruption that can be caused to the collection of pods the Service matching
+	DisruptionBudget *DisruptionBudget `json:"disruptionBudget,omitempty"`
```

Comment:
@davidopp suggest to use Pod annotation. Why shouldn't we use Service and Job annotation? IMO, DisrutionBudget applies to a collection of Pods, so it should put in Service and Job(in their annotation in the first version).